src/MuTalk-Model/MTAnalysis.class.st

```diff
@@ -242,6 +247,16 @@ MTAnalysis >> packagesToMutate: aCollectionOfPackages [
 		                packageName asPackage definedClasses ]
 ]
 
+{ #category : 'computing' }
+MTAnalysis >> percentile: aPercentage for: aCollection [
+
+	| size index |
+	aCollection isEmpty ifTrue: [ ^ nil ].
```

Comment:
Auch, returning nil is not good here... Can we throw an exception?

Reply:
Yes good point, I forgot to avoid returning nil